nifi-commons/nifi-security-utils/src/main/java/org/apache/nifi/security/util/CertificateUtils.java

```diff
@@ -469,6 +472,12 @@ public static X509Certificate generateSelfSignedX509Certificate(KeyPair keyPair,
             // (2) extendedKeyUsage extension
             certBuilder.addExtension(Extension.extendedKeyUsage, false, new ExtendedKeyUsage(new KeyPurposeId[]{KeyPurposeId.id_kp_clientAuth, KeyPurposeId.id_kp_serverAuth}));
 
+            // (3) subjectAlternativeName extension. Include CN as a SAN entry if it exists.
+            final String cn = getCommonName(dn);
+            if(cn != null) {
```

Comment:
Recommend adjusting this check to use `StringUtils.isNotBlank()` and adding a space after `if`:
```suggestion
            if (StringUtils.isNotBlank(cn)) {
```